zeroconf/test.py

```diff
@@ -492,9 +492,9 @@ def verify_name_change(self, zc, type_, name, number_hosts):
         self.assertRaises(r.NonUniqueNameException, zc.register_service, info_service)
 
         # verify no name conflict https://tools.ietf.org/html/rfc6762#section-6.6
-        zc.register_service(info_service, cooperating_responders=True)
+        zc.register_service(info_service, cooperating_responders=True, broadcast_service=False)
 
-        zc.register_service(info_service, allow_name_change=True)
+        zc.register_service(info_service, allow_name_change=True, broadcast_service=False)
```

Comment:
Is there a reason to change any existing call to `broadcast_service=False`? If so, we may need a note in the commit message explaining why we're doing that.

Reply:
Yes, its unneeded and slows down the test since it has to wait to broadcast

Reply:
I updated the text above